projects/prosoft-components-demo/src/app/slider-demo/slider-demo.component.ts

```diff
@@ -1,5 +1,4 @@
-import { ChangeDetectionStrategy } from '@angular/core';
-import { ChangeDetectorRef, Component } from '@angular/core';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component } from '@angular/core';
```

Comment:
You should consider disabeling the linter-rule for the error in line 66 at least for the demo project.

Reply:
This lint rule should actually be an error and these lines of code need to be reworked in a future PR. We changed it to warn in the angular 13 as we planned to bring a broad update to the demo, but this didn't happen up until now. :(